src/base/bittorrent/cachestatus.cpp

```diff
@@ -50,7 +50,11 @@ qreal CacheStatus::readRatio() const
 
 int CacheStatus::jobQueueLength() const
 {
+#if LIBTORRENT_VERSION_NUM < 10100
     return m_nativeStatus.job_queue_length;
+#else
+    return m_nativeStatus.queued_jobs;
```

Comment:
>@arvidn Do I chose an alternative right? Do they mean the same thing? Or maybe use session_settings::num_jobs instead?
>>The session_settings value is not a performance counter, it's the setting and it will only change if the client changes it.

Sorry it was a typo. I mean `cache_status::num_jobs`.